### PR TITLE
sales bar chart generator

### PR DESCRIPTION
**Pull request by deeworld**

### CODE REVIEW

File: sales bar chart generator/src/sales/bar/chart/generator/SalesBarChartGenerator.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        sales3=keyboard.nextInt();
+        System.out.print("Enter today's sales for store 4: ");
+        sales4=keyboard.nextInt();
+        System.out.print("Enter today's sales for store 5: ");
+        sales5=keyboard.nextInt();
+        System.out.println("SALES BAR CHART");
+        System.out.print("Store 1:");
+        for (int i=0; i<sales1; i+=100)
+
+            {
```
Comment (khatchad) [inline sales bar chart generator/src/sales/bar/chart/generator/SalesBarChartGenerator.java:37]: Correct formatting needed here.

File: sales bar chart generator/src/sales/bar/chart/generator/SalesBarChartGenerator.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                System.out.print("*");
+                //System.out.println();
+}
+                        System.out.println();
+                System.out.print("Store 5:");
+        for (int i=0; i<sales5; i+=100)
+
+            {
+                System.out.print("*");
+                //System.out.println();
```
Comment (khatchad) [inline sales bar chart generator/src/sales/bar/chart/generator/SalesBarChartGenerator.java:71]: Also, please do not submit code with commented statements.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+    Enter today's sales for store 3: 1800 [Enter]
+    Enter today's sales for store 4: 800 [Enter]
+    Enter today's sales for store 5: 1900 [Enter]
+    
+    SALES BAR CHART
+    Store 1: **********
+    Store 2: ************
+    Store 3: ******************
+    Store 4: ********
+    Store 5: *******************
```
Comment (khatchad) [inline README.md:15]: Why was this file changed?

Comment (deeworld) [inline README.md:15]: All I did was source format it and moved the commented statues. I didn't change anything else.

**Comment by khatchad** [comment]
Well, there's two java files here. Please delete the one you don't want me to grade and resubmit. Then, reopen the pull request.

**Comment by deeworld** [comment]
The one that says final hw sales bar chart with final corrections is the final one. It's basically the same as the first one but I did source format and removed the comments statues like you told me. I will try and delete the one with commented statues

**Comment by deeworld** [comment]
So I should delete the repository than commit it again and do a pull request?

**Comment by deeworld** [comment]
I deleted the repository and submitted it again.

**Comment by khatchad** [comment]
No need to delete the repository. You just need to delete the java file you don't want me to grab. You can even do that directly in GitHub if you wish. https://github.com/blog/1545-deleting-files-on-github
